src/Makefile.stable.toml

```diff
@@ -219,6 +228,10 @@ category = "Documentation"
 
 [tasks.pre-build]
 category = "Build"
+dependencies = [
```

Comment:
lets remove this block. no longer needed

Reply:
Done locally but forgot to stage it 😖